src/main/java/io/cryostat/agent/ConfigModule.java

```diff
@@ -159,6 +165,63 @@ public static Config provideConfig() {
         }
     }
 
+    @Provides
+    @Singleton
+    @Named(CRYOSTAT_AGENT_TRUSTSTORES)
+    public static List<TruststoreConfig> provideTruststoreConfigs(Config config) {
+        Map<Integer, TruststoreConfig> truststoreMap = new HashMap<>();
+
+        List<String> propertyNames =
+                StreamSupport.stream(config.getPropertyNames().spliterator(), false)
+                        .filter(e -> e.startsWith(CRYOSTAT_AGENT_TRUSTSTORES))
+                        .collect(Collectors.toList());
+
+        for (String name : propertyNames) {
+
+            String[] parts = name.split("\\.");
+            if (parts.length < 3) {
+                log.error(
+                        "Invalid truststore config property name format: {}. Rename to"
+                                + " 'truststore.cert[CERT_NUMBER].CERT_PROPERTY",
+                        name);
```

Comment:
Should there be a `continue;` here or something so that the invalid config gets skipped in processing?